README.md

```diff
@@ -11,14 +11,19 @@ This repository allows you to spin up a local test environment for the ``sdx-``
  - python3
 
 ### Getting started
+Make a projects directory e.g. `/Users/[NAME]/projects`
+In the ``projects`` make a directory named "sdx". this is where the SDX services will be.
 
-Export a ``SDX_HOME`` environment variable. This should point at a folder into
-which you wish to clone the rest of the services (e.g. ``/home/my-user/sdx``)
+Export a ``SDX_HOME`` environment variable.
```

Comment:
I'd change this to:
There needs to be an SDX_HOME environment variable pointed to where you will have all the SDX repos checked out to.  To do this, add   `export SDX_HOME=$HOME/projects/sdx` (the location is an example but you can have it checked out anywhere you want) to your .bash_profile (if using bash).